Sources/PointFree/Account/Account.swift

```diff
@@ -41,12 +41,12 @@ private func fetchAccountData<I>(
   let ownerSubscription = Current.database.fetchSubscriptionByOwnerId(user.id)
     .mapExcept(requireSome)
 
-  let owner = ownerSubscription
+  let subscription = userSubscription <|> ownerSubscription
+
+  let owner = subscription
```

Comment:
The "owner" supplied to the endpoint should always be the subscription owner and _not_ the owner of previous, cancelled subscriptions.